src/AzureAuth/Commands/Ado/CommandToken.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AzureAuth.Commands.Ado
+{
+    using McMaster.Extensions.CommandLineUtils;
+
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Office.Lasso.Telemetry;
+
+    /// <summary>
+    /// ADO Command for using either an ADO PAT or acquiring an AAD Access Token.
+    /// </summary>
+    [Command("token", Description = @"Return an AzureDevops PAT from an env var or authenticate the user via MSAL.
+This command should only be used by short running processes that can use either an Azure Devops PAT (long-lived)
+or an AAD Access Token (short-lived).")]

```

Comment:
I don't have an immediate suggestion, but is there any way for this help text to be... less long? Like maybe half the size? 😅

Reply:
This is only rendered on the help specifically for `azureauth ado --help` at which point I think it's important the `pat` and `token` comands clearly distinguish themselves. We don't have, at present other plans to better document this elsewhere and I dislike the thought of the docs not living on the command itself. Happy to wordsmith but i don't think this is place to try to save a few characters.

Reply:
I think we can make a big improvement to this, but I don't think it needs to be done now.

Reply:
I've tried to make this shorter, and dropped a link to our aka.ms/azureauth link. We can follow up this migration (or include) a PR revisiting the README and it's coverage of usage.